### DISCUSSION TITLE
Headless CMS example

### DISCUSSION THREAD

**Discussion by andrewMacmurray** [dillonkearns/elm-pages#197] [General]
I'm probably being a bit lazy here, but I wonder if it would be nice to have a canonical example of fetching a list of blog posts from a headless CMS.

The docs for constructing a `StaticHttp.Request` I find very clear, but I found it less obvious how to request / create a collection of pages that live alongside (or instead of) the local `content` folder and how these might map to urls

**Comment by andrewMacmurray** [comment]
lol I was being lazy, looks like it's mentioned here https://github.com/dillonkearns/elm-pages/issues/76 on the roadmap + a workaround to generate markdown files from a script

I'm putting together my own site using `elm-pages` + `contentful` so can make a cut down example from that

**Comment by dillonkearns** [comment]
I'm making good progress towards a version 2.0 of `elm-pages` now, and I've got it working with support for routes based on StaticHttp/data sources 🎉 Stay tuned, I'll be announcing something in the not-too-distant future! 😄 

I'd love to see your contentful demo!

**Comment by schoening** [comment]
I'm using elm-pages 2.0 with sanity.io. Still a WIP and learning experience but I am certainly willing to create a guide / example app if needed

**Comment by dillonkearns** [comment]
The 2.0 is now published, and I would really love to see any guides, examples, or other resources on this!

I went ahead and converted this Issue into a Discussion to reduce the clutter in the issues a little bit. Looking forward to seeing what cool resources we get going now with the latest elm-pages version!
